packages/components/src/local/organisms/admin-login/index.spec.tsx

```diff
@@ -4,6 +4,11 @@ import renderer from 'react-test-renderer'
 
 import AdminLogin from './index'
 
+jest.mock('react-redux', () => ({
+  ...jest.requireActual('react-redux'),
+  useSelector: jest.fn(() => [])
+}))
+
```

Comment:
Can you explain what this block of code does please @lebaphi ?

Reply:
in the `admin-login` component I use a hook `useSelector`, so when implement the unit test with jest, we should mock this hook to make the test passed because the jest will not know what is the `useSelector` is.